app/views/sections/index.html.erb

```diff
@@ -0,0 +1,100 @@
+<% require 'time' %>
+
+<div class = "menubody">
+
+    <h1>Food Menu for <%=Time.now.strftime("%A, %b %d")%></h1>
+    <div class = "center">
+
+      <h3 class = "toggle">~ CLICK TO HIDE MENU ITEMS ~</h3>
+      <h3 class = "toggle", style ="display: none">~ CLICK TO SHOW MENU ITEMS ~</h3>
+
+      <ul>
+        <% @sections.each do |section| %>
+        <li>
+          <a><h2><%= link_to section.name, section %></h2></a>
+        </li>
+          <%=section.description %>
+
+
+          <div class = "hide">
+           <ul>
+            <% @items.each do |item| %>
+              <% if item.menu_section_id == section.id %>
```

Comment:
This is a very logical way to do this, but if you use the association methods then you don't need to do the comparison of ids here. By putting `has_many :items` in the MenuSection class, it will allow you to use `.items` on a particular section to return an array of items with a matching menu_section_id

```html
<% section.items.each do |item|
  <li>
    <span class='item'><%= item.name %>...</span>
  </li>
<% end %>
```